torch/csrc/jit/passes/onnx.cpp

```diff
@@ -206,15 +220,15 @@ void ToONNX(std::shared_ptr<tracer::TracingState>& state) {
         cloneNode(node);
       }
     IR_ELSEIFM(PythonOp)
-      auto pyobj = py::handle(value->pyobj.get());
-      if (py::hasattr(pyobj, "symbolic")) {
-        auto outputs = callPySymbolicMethod(value);
-        setOutputs(value->name(), node, outputs);
-      } else {
+      callPySymbolicMethod(value);
+    IR_ELSE()
+      if (node->kind() == kUndefined) {
+        // Undefined nodes get passed into Convolution, but then they are
+        // removed.  We'll test for leftover Undefined in export.cpp
         cloneNode(node);
+      } else {
+        callPySymbolicFunction(node);
```

Comment:
It seems to me that Undefined nodes don't have symbolic anyway, so why do you need this conditional (now that `callPySymbolicFunction` is a clone for nodes that don't have it).

Reply:
If we encounter a node that has no symbolic, we raise a warning in this case. I added a special case for Undefined to prevent this warning from occurring, since Undefineds legitimately occur when processing Conv.

Reply:
dumb question: why do we have Undefineds for conv?